src/main/log/Resources/modules/administration/logs/components/menu.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import {PropTypes as T} from 'prop-types'
+import omit from 'lodash/omit'
+
+import {trans} from '#/main/app/intl/translation'
+import {LINK_BUTTON} from '#/main/app/buttons'
+import {Toolbar} from '#/main/app/action/components/toolbar'
+import {MenuSection} from '#/main/app/layout/menu/components/section'
+
+const LogsMenu = (props) =>
+  <MenuSection
+    {...omit(props, 'path')}
+    title={trans('dashboard', {}, 'logs')}
```

Comment:
Should be the name of the tool from `tools` translation domain (which is also used elsewhere) `trans('logs', {}, 'tools')`